bin/hermes

```diff
@@ -98,6 +99,54 @@ def request_post(path, json):
     return response
 
 
+def print_labor(labor, fates):
+    """Print the details of a labor to standard out
+
+    Args:
+        labor: the labor to print
+        fates: all the fates so we can give suggestions
+    """
+    creation_time = parser.parse(labor["creationTime"])
+    creation_time = creation_time.replace(tzinfo=tz.tzutc())
+    creation_time = creation_time.astimezone(tz.tzlocal())
+    if labor.get("targetTime", None):
```

Comment:
He's checking that the value is truthy, not just that it exists. At least that was the case in the code where I suggested this.